src/pudl/output/ferc1.py

```diff
@@ -110,7 +110,7 @@
                 null_calculated_value_frequency=1.0,
             ),
             xbrl_factoid=MetricTolerances(
-                error_frequency=0.028,
+                error_frequency=0.035,
                 relative_error_magnitude=0.019,
                 null_calculated_value_frequency=1.0,
             ),
```

Comment:
adding ... data!... into the liabilities table for many years, we the calc checks got a little worse for `retained_earnings`. Still pretty good compared to the other inter-table calcs.